### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jdbohrman**
Update documentation for `k3os.k3s_args` to better explain syntax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -573,14 +573,20 @@ k3os:
 ### `k3os.k3s_args`
 
 Arguments to be passed to the k3s process.  The arguments should start with `server` or `agent`
-to be valid.
+to be valid. `k3s_args` is an exec-style (aka uninterpreted) argument array, this means that k3os is invoking k3s in two seperate arguments like so:
 
-Example
+```bash
+#!/bin/sh
+exec "k3s" "server" "--cluster-cidr 10.107.0.0/23" "--service-cidr 10.107.1.0/23"
```
Comment (dweomer) [inline README.md]: `exec "k3s" "server" "--cluster-cidr 10.107.0.0/23" "--service-cidr 10.107.1.0/23"`

This is exactly the kind of thing folks have been tripping over. It should be either of the below:
- `exec "k3s" "server" "--cluster-cidr=10.107.0.0/23" "--service-cidr=10.107.1.0/23"`
- `exec "k3s" "server" "--cluster-cidr" "10.107.0.0/23" "--service-cidr" "10.107.1.0/23"`

Comment (dweomer) [reply README.md]: This is incorrect here, but correct below.

File: README.md [hunk 2]
```diff
@@ -573,14 +573,20 @@ k3os:
...
+```
+
+Example:
 ```yaml
 k3os:
-  k3s_args:
+   k3s_args:
   - server
-  - "--disable-agent"
+  - "--cluster-cidr" "10.107.0.0/23"
```
Comment (dweomer) [inline README.md]: These are two separate args and therefore should be two separate array entries. Please see the above comment.

File: README.md (hunk 1 above)
Comment (jdbohrman) [reply README.md]: Ah ok missed that part! I'll make that edit, thanks man :)

File: README.md (hunk 2 above)
Comment (dweomer) [reply README.md]: This commend applies here too: https://github.com/rancher/k3os/pull/373#discussion_r375966935

**Pull request merged**